src/test/java/teammates/test/cases/browsertests/InstructorFeedbackResultsPageUiTest.java

```diff
@@ -481,32 +500,36 @@ private void testPanelsCollapseExpand() {
         resultsPage.clickGroupByTeam();
 
         resultsPage.displayByGiverRecipientQuestion();
+        clickAjaxLoadedPanelAndWaitForExpansion("panelHeading-section-0-1", "ajax_auto");
         resultsPage.clickSectionCollapseStudentsButton();
         resultsPage.waitForSectionStudentPanelsToCollapse();
 
         resultsPage.displayByQuestion();
 
         ______TS("Typical case: panels expand/collapse");
 
+        assertEquals("Expand Questions", resultsPage.collapseExpandButton.getText());
+        assertEquals("Expand all panels. You can also click on the panel heading to toggle each one individually.",
+                     resultsPage.collapseExpandButton.getAttribute("data-original-title"));
+        resultsPage.verifyResultsHidden();
+
+        clickCollapseExpandButtonAndWaitForPanelsToRespond(true);
         assertEquals("Collapse Questions", resultsPage.collapseExpandButton.getText());
         assertEquals("Collapse all panels. You can also click on the panel heading to toggle each one individually.",
                      resultsPage.collapseExpandButton.getAttribute("data-original-title"));
         resultsPage.verifyResultsVisible();
 
-        resultsPage.clickCollapseExpand();
-        resultsPage.waitForPanelsToCollapse();
+        clickCollapseExpandButtonAndWaitForPanelsToRespond(false);
         assertEquals("Expand Questions", resultsPage.collapseExpandButton.getText());
         assertEquals("Expand all panels. You can also click on the panel heading to toggle each one individually.",
                      resultsPage.collapseExpandButton.getAttribute("data-original-title"));
         resultsPage.verifyResultsHidden();
 
-        resultsPage.clickCollapseExpand();
-        resultsPage.waitForPanelsToExpand();
+        clickCollapseExpandButtonAndWaitForPanelsToRespond(true);
```

Comment:
That is no longer the case after this PR : [Instructor view results: do not auto-load 3 questions #6568](https://github.com/TEAMMATES/teammates/pull/6608). Now the pages are not auto expanded at the start. The reason I expanded all panels here is because this test is to **testPanelsCollapseExpand** and I think clicking on collapse expand button is needed here.